tests/std/tests/P0295R0_gcd_lcm/test.compile.pass.cpp

```diff
@@ -36,7 +36,7 @@ static_assert(gcd(1073741824, 536870912) == 536870912);
 static_assert(gcd(1073741824, -536870912) == 536870912);
 static_assert(gcd(-1073741824, 536870912) == 536870912);
 static_assert(gcd(int_max, int_max) == int_max);
-static_assert(gcd(int_min, int_max) == 1);
+// gcd(int_min, int_max) -> undefined behavior

```

Comment:
Such constant evaluation failure can be `static_assert`'d with some SFINAE tricks. Although that's really tricky and only working in debug mode...

Reply:
I prefer to keep tests simple, and only test positive cases